sales.html

```diff
@@ -13,6 +13,29 @@
     <h2>Here there be cookie tables!</h2>
     <table id="cookieTable"></table>
   </section>
+  <section>
+    <form id="add_store_form">
+      <legend>Here Be Another Cookie Store</legend>
+      <fieldset>
+        <label for="shop_location">Shop Location</label>
+        <input id="shop_location" name="shop_location" type="text">
+      </fieldset>
+      <fieldset>
+        <label for="min_cust_per_hour">Minumum Customers per Hour</label>
+        <input id="min_cust_per_hour" name="min_cust_per_hour" type="text">
+      </fieldset>
+      <fieldset>
+        <label for="max_cust_per_hour">Maximum Customers per Hour</label>
+        <input id="max_cust_per_hour" name="max_cust_per_hour" type="text">
+      </fieldset>
+      <fieldset>
+        <label for="avg_cookies_per_hour">Average Cookies per Hour</label>
+        <input id="avg_cookies_per_hour" name="avg_cookies_per_hour" type="text">
+      </fieldset>
```

Comment:
Well this is interesting. A different fieldset for every input field. It's not a ding on you, more just an observation.